torch/nn/functional.py

```diff
@@ -63,13 +72,23 @@
 Note:
     {cudnn_reproducibility_note}
 """.format(**reproducibility_notes, **tf32_notes) + r"""
+
+    Padding mode 'valid' is the same as no padding. 'same' pads the input so
```

Comment:
Let's be careful to refer to "padding" vs. "padding mode", since even though "padding_mode" is not a parameter of the functional convolution it is a parameter of the module, and we don't want users to confuse the two.